clients/cloud/python/README.md

```diff
@@ -17,6 +17,24 @@ sasl.username=<api-key-id>
 sasl.password=<secret-access-key>
 ```
 
+## Configure SSL trust store
+
+Depending on your operating system or Linux distro you may need to take extra steps to set up the SSL CA root certificates.
+
+### CentOS
+
+```bash
+$ sudo yum reinstall ca-certificates
+```
+
+Add the following property to the config dict in the `Producer` and `AdminClient` objects in `producer.py`:
+
+```
+ssl.ca.location='/etc/ssl/certs/ca-bundle.crt'
```

Comment:
This needs to be in Python dict format:
`'ssl.ca.location': '/etc/ssl/certs/ca-bundle.crt'`

For reference:

```python
    # Create Producer instance
    p = Producer({
           'bootstrap.servers': conf['bootstrap.servers'],
           'sasl.mechanisms': 'PLAIN',
           'security.protocol': 'SASL_SSL',
           'sasl.username': conf['sasl.username'],
           'sasl.password': conf['sasl.password'],
           'ssl.ca.location': '/etc/ssl/certs/ca-bundle.crt'
    })
```

Reply:
Fixed in https://github.com/confluentinc/examples/pull/144/commits/e39b5937f3602243afc29b295c83f39253165f49